src/pages/Integrations/SplunkSetup/useSplunkSetup.tsx

```diff
@@ -33,26 +33,15 @@ Steps:
 ["ff59b502-da25-4297-bd88-6934ad0e0d63"] <<- Behavior group ID
 */
 
-import { useClient } from 'react-fetching-library';
-
 import { useGetAllEventTypes } from '../../../services/GetEventTypes';
-import { useGetAnyBehaviorGroupByNotification } from '../../../services/Notifications/GetBehaviorGroupByNotificationId';
-import { useGetBundleByName } from '../../../services/Notifications/GetBundles';
-import { linkBehaviorGroupAction } from '../../../services/Notifications/LinkBehaviorGroup';
-import { useSaveBehaviorGroupMutation } from '../../../services/Notifications/SaveBehaviorGroup';
-import { useUpdateBehaviorGroupActionsMutation } from '../../../services/Notifications/UpdateBehaviorGroupActions';
 import { useSaveIntegrationMutation } from '../../../services/useSaveIntegration';
 import {
   Integration,
   IntegrationCamel,
   IntegrationType,
   NewIntegrationTemplate,
 } from '../../../types/Integration';
-import {
-  BehaviorGroup,
-  BehaviorGroupRequest,
-  UUID,
-} from '../../../types/Notification';
+import { UUID } from '../../../types/Notification';
 
 export const SPLUNK_GROUP_NAME = 'SPLUNK_INTEGRATION';
 export const SPLUNK_INTEGRATION_NAME = 'SPLUNK_AUTOMATION';
```

Comment:
Please remove the `SPLUNK_BEHAVIOR_GROUP_NAME` constant.

Reply:
Done